requirements.txt

```diff
@@ -1,6 +1,6 @@
-dj-database-url==0.5.0
-Django==4.0.10
-gunicorn==20.1.0
-psycopg2-binary==2.9.3
+Django==4.2.3
+dj-database-url==2.0.0
+gunicorn==21.2.0
+psycopg2-binary==2.9.6

```

Comment:
Django 4.2 supports psycopg (3) but railway couldn't find it. So not updating to latest.